daisy/step_includeworkflow.go

```diff
@@ -33,7 +33,7 @@ type IncludeWorkflow struct {
 }
 
 func (i *IncludeWorkflow) populate(ctx context.Context, s *Step) DError {
-	if i.Path != "" {
+	if i.Path != "" && i.Workflow == nil {
```

Comment:
For posterity, add a comment on when `i.Workflow` could be non nil and why this check is needed.

Reply:
Done!